src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const PRICE = 40;
+
+  const TOTAL_PRICE = days * PRICE;
```

Comment:
it's not a static variable like LONG_TERM. We calculate it. We should write it in lowercase [totalPrice]